pkg/client/restclient/config.go

```diff
@@ -45,10 +46,14 @@ const (
 // Config holds the common attributes that can be passed to a Kubernetes client on
 // initialization.
 type Config struct {
-	// Host must be a host string, a host:port pair, or a URL to the base of the apiserver.
-	// If a URL is given then the (optional) Path of that URL represents a prefix that must
-	// be appended to all request URIs used to access the apiserver. This allows a frontend
-	// proxy to easily relocate all of the apiserver endpoints.
+	// Scheme is the name of protocol used for communication with the server (http or
+	// unix).
+	Scheme string
+	// Host must be a host string, a host:port pair, a URL or a path to unix socket to the
+	// base of the apiserver. If a URL is given then the (optional) Path of that URL
+	// represents a prefix that must be appended to all request URIs used to access the
+	// apiserver. This allows a frontend proxy to easily relocate all of the apiserver
```

Comment:
The godoc for Config says it should be supported, when I wrote it, it supported URLs.  If it doesn't support URLs, why did that break?